### DISCUSSION TITLE
Compare and benchmark two GC implemetations

### DISCUSSION THREAD

**Discussion by dtrendafilov** [SofiaCPP/IPL#108] [Projects]
Implement at least two GC algorithms and use the data from the [test bench](https://github.com/SofiaCPP/IPL/tree/master/gctest) to compare them.

**Comment by dtrendafilov** [comment]
A test app for the GC https://github.com/SofiaCPP/IPL/tree/feature/gctest/gctest

**Comment by dimitroffangel** [comment]
Аз да си маркирам едно място за проекта в крайна смета ^ ^
Цел в проекта ще вдигна първото двата базови модела за GC (reference counting, mark-sweep collecting)
После алгоритъма на Cheney за garbage collector (тоест разделям хийпа на две части копирам това, което ще ми трябва, останалото го убивам)
Съответно след като ги направя ще ги сравня с няколко мои алокатора (SLAB, Buddy, Pool-ish allocator) и със стандартния.

**Comment by RadoslavSV** [comment]
Ще правя този проект. (Радослав Велков, СИ, 62528)

**Reply by dtrendafilov** [reply]
:+1:

**Comment by petkobankov** [comment]
Ще правя този проект. Петко Банков 81854

**Comment by GeorgiMinchev1** [comment]
Ще правя този проект - Георги Минчев, 81906.

**Comment by FilipStoyanov** [comment]
Ще правя този проект - Филип Стоянов 81950

**Comment by haki-1** [comment]
Ще правя този проект.
